### PR TITLE
Adding method in API to get entity relationships

### PR DESCRIPTION
**Pull request by Mun1z**
**Depends on #115 .**

Adding method in ML-Git API, which aims to list relationships of a specific entity. This method uses the Github API.

```
def get_entity_relationships(self, entity_name, metadata_repo_name, export_type=FileType.JSON.value, export_path=None):
    """Get a list of relationships for an entity.

    Args:
        entity_name (str): The name of the entity you want to get the linked entities.
        metadata_repo_name (str): The repository name where the metadata is located in GitHub.
        export_type (str): Choose the format of the return [default: json].
        export_path (str): Set the path to export metrics to a file.

    Returns:
        list of EntityVersionRelationships.
    """
```

**Usage example:** 
```python
from ml_git import api
github_token = ''
api_url = 'https://api.github.com'
manager = api.init_entity_manager(github_token, api_url)

relationships = manager.get_entity_relationships('entity_name', metadata_repo_name='user/metadata_repository')
```

The variable `linked_entities ` should look something like:
```
>>> manager.get_entity_relationship('model-ex', 'user/mlgit-models')
 {​'model-ex': [​, {​
  "version": 2,
  "tag": "test__model-ex__2",
  "relationships": [
    {​
      "tag": "test__dataset-ex__1",
      "name": "dataset-ex",
      "version": "1",
      "entity_type": "dataset"
    }​,
    {​
      "tag": "test__labels-ex__2",
      "name": "labels-ex",
      "version": "2",
      "entity_type": "labels"
    }​
  ]
}​, {​
  "version": 1,
  "tag": "test__model-ex__1",
  "relationships": [
    {​
      "tag": "test__dataset-ex__1",
      "name": "dataset-ex",
      "version": "1",
      "entity_type": "dataset"
    }​,
    {​
      "tag": "test__labels-ex__1",
      "name": "labels-ex",
      "version": "1",
      "entity_type": "labels"
    }​
  ]
}​]}​
 
```

### CODE REVIEW

File: docs/api/README.md [hunk 1]
```diff
@@ -297,6 +297,18 @@ class EntityManager:
         Returns:
             list of LinkedEntity.
         """
+    def get_entity_relationships(self, entity_name, metadata_repo_name, export_type=FileType.JSON.value, export_path=None):
+        """Get a list of relationships for an entity.
+
+        Args:
+            entity_name (str): The name of the entity you want to get the linked entities.
+            metadata_repo_name (str): The repository name where the metadata is located in GitHub.
```
Comment (lucasncabral) [inline docs/api/README.md:305]:
```suggestion
            metadata_repo_name (str): The repository name where the entity metadata is located in GitHub.
```

File: ml_git/relationship/utils.py [hunk 2]
```diff
@@ -31,3 +37,36 @@ def get_repo_name_from_url(repository_url: str) -> str:
...
+                'to_type': to_entity.entity_type
+            })
+    file_name = '{}_relationships.{}'.format(entity_name, FileType.CSV.value)
+
+    if export_path is None:
+        with tempfile.TemporaryDirectory() as tempdir:
+            file_path = os.path.join(tempdir, file_name)
+            create_csv_file(file_path, csv_header, formatted_data)
+            with open(file_path) as csv_file:
+                return io.StringIO(csv_file.read())
```
Comment (lucasncabral) [inline ml_git/relationship/utils.py:66]: This code snippet appears to be the same as lines 68-72. If this is the scenario, please extract to a method.

File: docs/api/README.md [hunk 3]
```diff
@@ -390,6 +402,22 @@ class Storage:
     """
 ```
 </details>
+<details markdown="1">
+<summary><code> EntityVersionRelationships </code></summary>
+<br>
+
+```python
+class Relationship:
```
[the remark this answers is not in the document]
Comment (lucasncabral) [reply docs/api/README.md]: Documentation was outdated. Was updated in 197fbeb.

**Review by lucasncabral — APPROVED** [review]

**Review by gilMars — APPROVED** [review]

File: docs/api/README.md [hunk 4]
```diff
@@ -390,6 +402,22 @@ class Storage:
     """
 ```
 </details>
+<details markdown="1">
+<summary><code> EntityVersionRelationships </code></summary>
+<br>
+
+```python
+class EntityVersionRelationships:
+    """Class that's represents the relationships of an ml-entity in a specified version.
```
Comment (andersonpablosilva) [inline docs/api/README.md:411]:
```suggestion
    """Class that represents the relationships of an ml-entity in a specified version.
```

Please fix all similar errors.

Comment (lucasncabral) [reply docs/api/README.md]: The message was updated in 1990ba0.

**Review by andersonpablosilva — APPROVED** [review]

**Review by gpretto — APPROVED** [review]

**Pull request merged**